app/main.py

```diff
@@ -41,7 +43,7 @@ def create_tables(engine, psql_environment):
 
 from app.routers import (  # noqa: E402
     about_us, agenda, calendar, categories, celebrity, credits,
-    currency, dayview, email, event, export, four_o_four, friendview,
+    currency, dayview, email, event, export, four_o_four, features, friendview,
```

Comment:
The imports are not ordered. Use prehooks to automatically order them (see README)

Reply:
Didn't find what your talking about' but  i ordered it.